packages/core/src/typeschema/types.ts

```diff
@@ -373,7 +373,7 @@ class StructureDefinitionParser {
         } while (this.backboneContext && !pathsCompatible(this.backboneContext.path, element?.path));
       } else {
         this.innerTypes.push(this.backboneContext.type);
-        delete this.backboneContext;
+        this.backboneContext = undefined;
```

Comment:
Out of curiosity, was this required for something, or more of a stylistic choice?

Reply:
<img width="912" alt="Screenshot 2024-04-01 at 9 37 45 AM" src="https://github.com/medplum/medplum/assets/1592008/8eaca642-d749-4679-8736-17cb76698fb0">
Not really stylistic, `delete` is super slow because it causes changes in `HiddenClass` structure